.travis.yml

```diff
@@ -1,5 +1,10 @@
 sudo: required
 
+addons:
```

Comment:
`jq` should be available in the images by default. I thought I'd seen it mentioned in the docs but I'm unable to find it. However, this comment confirms it https://github.com/travis-ci/travis-ci/issues/7954#issuecomment-310527961